logger/syslogd/syslogd.go

```diff
@@ -112,7 +26,14 @@ func Listen(exitChan, cleanupDone chan bool, drainChan chan string, bindAddr str
 	}
 	// Create a server with one handler and run one listen goroutine
 	s := syslog.NewServer()
-	h := newHandler()
+	var h *handler.Handler
+	if HandlerType == "standard" {
+		fmt.Println("Using standard handler for syslog")
+		h = handler.StandardHandler(LogRoot)
+	} else {
+		fmt.Println("Using ring buffer handler for syslog")
+		h = handler.RingBufferHandler(RingBufferSize, WebServicePort)
```

Comment:
Separation of concerns... let the ring buffer's package worry about getting the configuration that is specific to it instead of having the `syslogd` package worry about it when the ring buffer may not even be in use.